Dockerfile

```
@@ -64,4 +64,4 @@ COPY app/ app/
 COPY .streamlit/ .streamlit/
 USER 1000
 EXPOSE 8080
-CMD ["streamlit", "run", "app/main.py"]
+CMD ["poetry", "run", "streamlit", "run", "app/data_visualization.py", "--server.port=8080", "--server.address=0.0.0.0"]
```

```suggestion
CMD ["streamlit", "run", "app/data_visualization.py", "--server.port=8080", "--server.address=0.0.0.0"]
```

prod 用 イメージに poetry を入れていないのでこれでお願いします

---

@taisei173263 

+ plotly など追加のパッケージを使っていると思うので、`poetry add パッケージ名` の実行もお願いします！